@here/olp-sdk-dataservice-read/lib/DataStoreDownloadManager.ts

```diff
@@ -96,7 +96,8 @@ export class DataStoreDownloadManager implements DownloadManager {
             const response = await fetchFunction(url, init);
             // tslint:disable-next-line: no-magic-numbers
             if (response.status !== 503 || retryCount >= maxRetries) {
-                if (response.status === 200) {
+                // tslint:disable-next-line: no-magic-numbers
```

Comment:
Please create  enum with status codes and remove ` // tslint:disable-next-line: no-magic-numbers`

Reply:
Done